roxctl/netpol/generate/generate.go

```diff
@@ -0,0 +1,136 @@
+package generate
+
+import (
+	"os"
+
+	npguard "github.com/np-guard/cluster-topology-analyzer/pkg/controller"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/npg"
+	"github.com/stackrox/rox/roxctl/common/printer"
+)
+
+// NetpolGenerateOptions represents input parameters to NetpolGenerateCmd
+type NetpolGenerateOptions struct {
+	StopOnFirstError      bool
+	TreatWarningsAsErrors bool
+	OutputFolderPath      string
+	OutputFilePath        string
+	RemoveOutputPath      bool
+}
+
+// NetpolGenerateCmd represents NP-Guard functionality for generating network policies
+type NetpolGenerateCmd struct {
```

Comment:
Is the split of methods for this struct defined here and within `generate/cta.go` really required?

This isn't something we typically do (splitting the implementation of an interface across multiple files) and avoid doing so for readability.